openghg/store/_oneD_timeseries.py

```diff
@@ -0,0 +1,339 @@
+                - "combine" - replace and insert new data into current timeseries
+            save_current: Whether to save data in current form and create a new version.
+                - "auto" - this will depend on if_exists input ("auto" -> False), (other -> True)
+                - "y" / "yes" - Save current data exactly as it exists as a separate (previous) version
+                - "n" / "no" - Allow current data to updated / deleted
+            overwrite: Deprecated. This will use options for if_exists="new".
+            force: Force adding of data even if this is identical to data stored.
+            compressor: A custom compressor to use. If None, this will default to
+                `Blosc(cname="zstd", clevel=5, shuffle=Blosc.SHUFFLE)`.
+                See https://zarr.readthedocs.io/en/stable/api/codecs.html for more information on compressors.
+            filters: Filters to apply to the data on storage, this defaults to no filtering. See
+                https://zarr.readthedocs.io/en/stable/tutorial.html#filters for more information on picking filters.      Returns:
+            dict: Dictionary of datasource UUIDs data assigned to
+        """
+        from openghg.types import OneD_types
+
+        from openghg.util import (
+            clean_string,
+            load_oned_parser,
+            check_if_need_new_version,
+        )
+
+        species = clean_string(species)
+        source = clean_string(source)
+        domain = clean_string(domain)
+
+        if overwrite and if_exists == "auto":
+            logger.warning(
+                "Overwrite flag is deprecated in preference to `if_exists` (and `save_current`) inputs."
+                "See documentation for details of these inputs and options."
+            )
+            if_exists = "new"
+
+        # Making sure new version will be created by default if force keyword is included.
+        if force and if_exists == "auto":
+            if_exists = "new"
+
+        new_version = check_if_need_new_version(if_exists, save_current)
+
+        filepath = Path(filepath)
+
+        try:
+            source_format = OneD_types[source_format.upper()].value
+        except KeyError:
+            raise ValueError(f"Unknown data type {source_format} selected.")
+
+        # Load the data retrieve object
+        parser_fn = load_oned_parser(source_format=source_format)
+
+        _, unseen_hashes = self.check_hashes(filepaths=filepath, force=force)
+
+        if not unseen_hashes:
+            return {}
+
+        filepath = next(iter(unseen_hashes.values()))
+
+        if chunks is None:
+            chunks = {}
+
+        # Define parameters to pass to the parser function
+        # TODO: Update this to match against inputs for parser function.
+        param = {
+            "filepath": filepath,
+            "species": species,
+            "domain": domain,
+            "source": source,
+            "data_type": "OneDTimeseries",
```

Comment:
Should match to whatever alternative `data_type` name is chosen (e.g. "flux_timeseries")